### PR TITLE
Use git submodules

### PR DESCRIPTION
**Pull request by abulte**
Instead of manual cloning, because why not? ;-)

This leads to a new directory layout because you can't add submodules inside already versioned directories.

Iterates on https://github.com/etalab/data.gouv.fr/issues/908

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[submodule "csvapi"]
+	path = csvapi
+	url = git@github.com:etalab/csvapi.git
+	branch = kafka-integration
+[submodule "udata-hydra"]
+	path = udata-hydra
+	url = git@github.com:etalab/udata-hydra.git
+[submodule "udata-analysis-service"]
+	path = udata-analysis-service
+	url = git@github.com:etalab/udata-analysis-service.git
```
Comment (maudetes) [inline .gitmodules]:
```suggestion
	url = git@github.com:etalab/udata-hydra.git
	branch = main
[submodule "udata-analysis-service"]
	path = udata-analysis-service
	url = git@github.com:etalab/udata-analysis-service.git
	branch = main
```

File: README.md [hunk 2]
```diff
@@ -51,7 +51,8 @@ After installing udata event orchestration package, you can run end-to-end tests
 
 Make sure you have your docker-compose with all services running and run:
 ```
-pytest
+pip install pytest
+pytest tests/
```
Comment (maudetes) [inline README.md:55]: I saw that I add added a `pytest.ini` with the testpaths but hadn't push.
But `pytest tests` may be more explicit and allow to run *all* tests instead also.

**Review by maudetes — APPROVED** [review]
I think you should modify the part on config files in the Readme to follow the new layout.

You could also add the comment to start all the different docker-compose.

**Comment by abulte** [comment]
@maudetes I backported the relevant README modifications from https://github.com/etalab/udata-event-orchestration/pull/7/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

**Comment by geoffreyaldebert** [comment]
Cool ! I never used gitmodules ;)

**Pull request merged**